src/components/Link1/Link1.tsx

```diff
@@ -0,0 +1,9 @@
+
+const Link1=()=>{
+    return(
+        <div>
+            Link1
+        </div>
+    )
+}
+export default Link1
```

Comment:
Ok, let's keep that page, but, at least, rename it to the ExamplePage and move to folder `pages`, not `components` (and update routes)

Reply:
delete all links on footer